crates/ruff/resources/test/fixtures/jupyter/not_json.ipynb

```diff
@@ -1 +1 @@
-broken "§=($/=(")
+broken "§=($/=()
```

Comment:
The lexer no longer marks in-balanced parentheses pairs as errors (because it also doesn't complain about `[{(]})` as unbalanced even though the parens are closed in the wrong order.